docs/call-numbers.md

```diff
@@ -0,0 +1,19 @@
+# Call Numbers
+
+Note: This documentation was migrated from https://docs.google.com/document/d/1bHvgfgyUmDXV7ROqEZFaRxJFEYhSSUbeQoQT883GMWg/edit#heading=h.oz6ax25g11u and needs to be updated
+
+## Generating browse lists
+
+To fully regenerate all browse lists:
+`OL_DB_PORT=5432 bundle exec rake browse:all && OL_DB_PORT=5432 bundle exec rake browse:load_all`
+
+Expected time: 5.5 - 6 hours.
+
+## Troubleshooting
+
+When browse lists fail to generate and would like to run the rake task before it is scheduled:
+ssh to `lib-pulindexer1`. Check the crontab and make sure the rake task for the call numbers runs on this box. If not then check `lib-pulindexer2` and `lib-pulindexer3`.
```

Comment:
catalog-indexer1, catalog-indexer2, catalog-indexer3. Currently the call_numbers task is deployed to run on catalog-indexer1 check the crontab to confirm.

Reply:
@hackartisan can you update the worker names? ^